job-dsls/jobs/kie/kogito/kogito_tooling_prerelease_branch_UMB_trigger.groovy

```diff
@@ -0,0 +1,91 @@
+/**
+ * Creates job that triggers when a new `<MAJOR>-<MINOR>-<PATCH>-prerelease` branch is
+   pushed to kogito-tooling repository and notifies QE of this by sending and UMB message.
+ */
+import org.kie.jenkins.jobdsl.Constants
+
+def repo = "kogito-tooling"
+def repoBranch = ''
+def ghOrgUnit = "kiegroup"
+def ghAuthTokenId = "kie-ci-token"
+def labelName = "kie-rhel7 && kie-mem16g"
+
+// creation of folder
+folder("KIE")
+folder ("KIE/kogito")
+folder ("KIE/kogito/kogito-tooling")
+
+def folderPath="KIE/kogito/kogito-tooling"
+
+String jobName = "${folderPath}/${repo}-prerelease-branch-UMB-trigger"
+
+job(jobName) {
+
+    description("Sends UMB message when new prerelease branch is pushed to kogito-tooling")
+
+    logRotator {
+        numToKeep(10)
+    }
+
+    parameters {
+        stringParam("sha1")
+    }
+
+    concurrentBuild()
+
+    properties {
+        ownership {
+            primaryOwnerId("mbiarnes")
+            coOwnerIds("mbiarnes")
+        }
+    }
+
+    jdk(javadk)
+
+    label(labelName)
+
+    triggers {
+         genericTrigger {
+                genericVariables {
+                    genericVariable {
+                        key("ref")
+                        value("\$.ref")
+                        regexpFilter("refs/head/[0-9]*.[0-9]*.[0-9]*-prerelease")
+                    }
+                    genericVariable {
+                        key("created")
+                        value("\$.created")
+                        regexpFilter("true")
+                    }
+                }
+                token(ghAuthTokenId)
+         }
+
+    }
+
+    wrappers {
+        timeout {
+            elastic(150, 3, 90)
+        }
+        timestamps()
+        colorizeOutput()
+
+        preBuildCleanup()
+    }
+
+    steps {
+        stage ('UMB message'){
+            steps {
+                    if (${ref} && ${created}) {
+                        def KOGITO_TOOLING_VERSION = ($ref ~= [0-9]*\.[0-9]*\.[0-9]*)[0][1]
```

Comment:
this is the only point where it is failing right now

Reply:
Yea, I need to `${ref}`  the variable

Reply:
and the operator is in wrong order